web/src/components/auth/SelfRegSnackbar.tsx

```diff
@@ -83,7 +90,13 @@ export const SelfRegSnackbar = (): ReactElement => {
               color: "#757575",
             }}
           >
-            <Icon className="usa-icon--size-4">close</Icon>
+            <Icon
```

Comment:
[dust] I'd ask design it the sizing change is intentional. I'd expect consistency especially since they are required to create a variation of the snackbar in the design library with the different icon size and they haven't.